libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -45,6 +50,19 @@ export class BookSearchComponent implements OnInit, OnDestroy {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    const snackBarRef = this._snackBar.open(
```

Comment:
preferred approach is to move the snackBar to readinglist-effects, so that the injecting same snackbar service to both components can be avoided